docs/proposals/buildstrategy-steps-resources.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Build Strategies steps resource limitations
+authors:
+  - "@qu1queee"
+  - "@xiujuan95"
+  - "@SaschaSchwarze0"
+  - "@zhangtbj"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-06-15
+last-updated: 2020-06-15
+status: implementable
+see-also:
+  - "/docs/proposals/buildstrategy-steps-resources.md"
+---
+
+# Build Steps Resource Limitations
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [x] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions [optional]
+
+> 1. This locks end-users to define different resource constrains for multiple Build Strategy steps. Build/ClusterBuildStrategy admins can still do this. Please keep in mind that in development
+  environments (_e.g. kind cluster, personal k8s cluster_) the end-user plays also the role of `strategy admin`, because he/she needs to deploy the controller and install the Build/ClusterBuildStrategy CRDs instances.
+  For production environments(_e.g. IKS, Openshift_), an end-user will not play the role of the `strategy admin`, so he/she knows nothing about the internals of the Build/ClusterBuildStrategy CRDs instances.
```

Comment:
I have difficulties with this section, somehow under an `Open questions` section, I am searching for a question. Are you maybe bringing up the conceptional question _Should end-users (=users that define build and buildrun but not (cluster) build strategy) have fine-grained control on resource definitions for the different build steps (containers) of a build strategy?_

Reply:
I refactored the sentence/section